NuRadioMC/SignalProp/analyticraytracing.py

```diff
@@ -1942,8 +1942,10 @@ class describing the index-of-refraction profile
             if True, use CPP implementation of minimization routines
             default: True if CPP version is available
 
+        compile_numba: bool (default: None)
+            Only relevant if `use_cpp` is False. If None, the default is True (if `use_cpp` is False).
```

Comment:
I'm not sure how I feel about `None` instead of `True`, usually explicit is better than implicit, but I guess there is some ambiguity because it gets overwritten by `use_cpp`